client/CPlayerInterface.cpp

```diff
@@ -572,7 +572,14 @@ void CPlayerInterface::garrisonsChanged(std::vector<const CGObjectInstance *> ob
 		auto * town = dynamic_cast<const CGTownInstance*>(object);
 
 		if (hero)
+		{
 			adventureInt->onHeroChanged(hero);
+
+			if(hero->inTownGarrison)
+			{
+				adventureInt->onTownChanged(hero->visitedTown);
+			}

```

Comment:
Personal opinion, so OK to keep but for 1-line code block brackets are unnecessary

Reply:
hmmm I would need to recheck coding guidelines, but at least back in the day there was rule that if code is part of a block (other than function-level block) then internal part should be wrapped